aspnetcore/migration/mvc.md

```diff
@@ -125,25 +119,24 @@ Note the lack of styling and menu items. We'll fix that in the next section.
 
 ## Static content
 
-In previous versions of  ASP.NET MVC, static content was hosted from the root of the web project and was intermixed with server-side files. In ASP.NET Core, static content is hosted in the *wwwroot* folder. You'll want to copy the static content from your old ASP.NET MVC app to the *wwwroot* folder in your ASP.NET Core project. In this sample conversion:
+In previous versions of ASP.NET MVC, static content was hosted from the root of the web project and was intermixed with server-side files. In ASP.NET Core, static content is hosted in the *wwwroot* folder. You'll want to copy the static content from your old ASP.NET MVC app to the *wwwroot* folder in your ASP.NET Core project. In this sample conversion:
 
 * Copy the *favicon.ico* file from the old MVC project to the *wwwroot* folder in the ASP.NET Core project.
 
 The old ASP.NET MVC project uses [Bootstrap](http://getbootstrap.com/) for its styling and stores the Bootstrap files in the *Content* and *Scripts* folders. The template, which generated the old ASP.NET MVC project, references Bootstrap in the layout file (*Views/Shared/_Layout.cshtml*). You could copy the *bootstrap.js* and *bootstrap.css* files from the ASP.NET MVC project to the *wwwroot* folder in the new project, but that approach doesn't use the improved mechanism for managing client-side dependencies in ASP.NET Core.
 
-In the new project, we'll add support for Bootstrap (and other client-side libraries) using [Bower](https://bower.io/):
-
-* Add a [Bower](https://bower.io/) configuration file named *bower.json* to the project root (Right-click on the project, and then **Add > New Item > Bower Configuration File**). Add [Bootstrap](http://getbootstrap.com/) and [jQuery](https://jquery.com/) to the file (see the highlighted lines below).
-
-  [!code-json[](mvc/sample/bower.json?highlight=5-6)]
-
-Upon saving the file, Bower will automatically download the dependencies to the *wwwroot/lib* folder. You can use the **Search Solution Explorer** box to find the path of the assets:
+In the new project, we'll add support for Bootstrap (and other client-side libraries) using CDN:
 
-![jquery assets shown in the Solution Explorer search results](mvc/_static/search.png)
+* Open **Views > Shared > _Layout.cshtml** and add the following link to css and script tags
```

Comment:
~~* Open **Views > Shared > _Layout.cshtml** and add the following link to css and script tags~~

```
* Open *Views/Shared/_Layout.cshtml* and add the following markup:
```